compiler/flops.cc

```diff
@@ -32,22 +32,66 @@ int64_t OutputSize(const Node& node) {
 }
 
 int64_t CalculateFlopsOfGemm(const Node& node) {
-    int64_t flops = node.output(0)->type().NumElements();
+    int64_t const out_size = node.output(0)->type().NumElements();
+    int64_t flops = out_size;
     flops *= node.input(0)->type().dims()[node.trans_a() ? 0 : 1];
+    if (node.alpha() != 1.0) {
+        flops += out_size;
+    }
+    if (node.beta() != 0.0) {
+        flops += out_size * node.input(2)->type().dims()[node.trans_a() ? 0 : 1];
+        if (node.beta() != 1.0) {
+            flops += out_size;
+        }
+    }
     return flops;
 }
 
 int64_t CalculateFlopsOfConv(const Node& node) {
-    int64_t bsize = node.input(0)->type().dims()[0];
-    int64_t ichan = node.input(0)->type().dims()[1];
-    int64_t kw = node.input(1)->type().dims()[2];
-    int64_t kh = node.input(1)->type().dims()[3];
-    int64_t ochan = node.output(0)->type().dims()[1];
-    int64_t ow = node.output(0)->type().dims()[2];
-    int64_t oh = node.output(0)->type().dims()[3];
+    Type const& x = node.input(0)->type();
+    Type const& w = node.input(1)->type();
+    Type const& y = node.output(0)->type();
+    int64_t bsize = x.dims()[0];
+    int64_t ichan = x.dims()[1];
+    int64_t kw = w.dims()[2];
```

Comment:
This is my fault, but it seems all of `w` and `h` are swapped :( The third dimension is `h` and the forth is `w`. This won't change the calculation though..